pageserver/src/tenant/timeline/walreceiver/connection_manager.rs

```diff
@@ -601,6 +604,20 @@ impl WalreceiverState {
                                     },
                                 });
                             }
+                            // If we have a candidate with the same commit_lsn as the current one, which is in the same AZ as pageserver,
```

Comment:
This function commented should be updated as well, it lists reasons for reconnection.